components/model.py

```diff
@@ -118,6 +118,10 @@ def delete_fixed_mounting(code, ay_sem):
     return database.delete_fixed_mounting(code, ay_sem)
```

Comment:
All functions missing doc-strings. Will not insist that doc-strings be added for now.